packages/api/cms-api/src/file-uploads/file-uploads-download.controller.ts

```diff
@@ -49,6 +49,15 @@ export function createFileUploadsDownloadController(options: { public: boolean }
 
         @Get(":hash/:id/:timeout")
         async download(@Param() { hash, ...params }: HashDownloadParams, @Res() res: Response, @Headers("range") range?: string): Promise<void> {
+            await this.downloadFile({ hash, ...params }, res, range, false);
+        }
+
+        @Get("preview/:hash/:id/:timeout")
+        async preview(@Param() { hash, ...params }: HashDownloadParams, @Res() res: Response, @Headers("range") range?: string): Promise<void> {
+            await this.downloadFile({ hash, ...params }, res, range, true);
+        }
+
+        private async downloadFile({ hash, ...params }: HashDownloadParams, res: Response, range?: string, forceInline = false) {
```

Comment:
Instead for forceInline you could set the content disposition like we do here: https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/dam/files/files.controller.ts#L265.